mbf_msgs/action/ExePath.action

```diff
@@ -26,17 +26,17 @@ uint8 PAT_EXCEEDED      = 103
 uint8 COLLISION         = 104
 uint8 OSCILLATION       = 105
 uint8 ROBOT_STUCK       = 106
-uint8 MISSED_GOAL       = 107
-uint8 MISSED_PATH       = 108
-uint8 BLOCKED_GOAL      = 109
-uint8 BLOCKED_PATH      = 110
+uint8 MISSED_GOAL       = 107  # The robot has overshot the goal and cannot reach it anymore
+uint8 MISSED_PATH       = 108  # The robot has diverged from the path and cannot rejoin it
+uint8 BLOCKED_GOAL      = 109  # There's an obstacle at the goal
+uint8 BLOCKED_PATH      = 110  # There's an obstacle on the path
```

Comment:
not 100% sure if it's a good idea to add comments, instead of letting the users to interpret the codes as they wish
I can remove if someone else thinks so